### PR TITLE
Add "The site collection "XXXX" cannot be updated or created

### PR DESCRIPTION
**Pull request by MSGRRojas**
This is to add what to do when a user receives this error

### CODE REVIEW

File: migration/mm-troubleshoot.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ This article describes how to resolve issues and errors you may experience when
 - [Error codes](#error-codes)
 - [Geo admins can't access full functionality of Migration Manager](#geo-admins-not-supported)
 - [Group-inherited SharePoint admins can't access full functionality of Migration Manager](#group-inherited-sharepoint-admins-unable-to-access-scans-and-migrations-tabs))
+- [The site collection "XXXX" cannot be created or updated] (#The-site-collection-"XXXX"-Cannot-be-created-or-updated)
```
Comment (JohanFreelancer9) [inline migration/mm-troubleshoot.md:40]:
```suggestion
- [The site collection "XXXX" cannot be created or updated](#The-site-collection-"XXXX"-Cannot-be-created-or-updated)
```

File: migration/mm-troubleshoot.md [hunk 2]
```diff
@@ -262,6 +263,10 @@ SharePoint admin roles that were created as a result of joining an Azure group a
 2. On the **Active users** page, select the user whose admin role you want to change. In the flyout pane, under **Roles**, select **Manage roles**.
 3. Select the admin role that you want to assign to the user. If you don't see the role you're looking for, select **Show all** at the bottom of the list.
 
+## The site collection "XXXX" cannot be created or updated
+
+The UPN referenced is invalid.  In order to create or update a OneDrive site collection, the referenced User Principal Name must be found in Azure Active Directory.  Please check your destination URL and validate the user exists in AAD.
```
Comment (JohanFreelancer9) [inline migration/mm-troubleshoot.md:268]:
```suggestion
The UPN referenced is invalid. In order to create or update a OneDrive site collection, the referenced User Principal Name must be found in Azure Active Directory. Check your destination URL and validate that the user exists in AAD.
```

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MSGRRojas, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**